docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -7,34 +7,61 @@ A {dfanalytics} configuration object has the following properties:
 
 `analysis`::
   (object) The type of analysis that is performed on the `source`. For example: 
-  `outlier_detection`.
+  `outlier_detection`. For more information, see <<dfanalytics-types>>.
+  
+`analyzed_fields`::
+  (object) You can specify both `includes` and/or `excludes` patterns. If 
+  `analyzed_fields` is not set, only the relevant fileds will be included. For 
+  example all the numeric fields for {oldetection}.
 
 `dest`::
-  (object) The destination configuration of the analysis, consisting of `index` 
-  and optionally `results_field` (`ml` by default). 
+  (object) The destination configuration of the analysis. For more information, 
+  see <<dfanalytics-dest-resources>>.
 
 `id`::
   (string) The unique identifier for the {dfanalytics-job}. This identifier can 
   contain lowercase alphanumeric characters (a-z and 0-9), hyphens, and 
   underscores. It must start and end with alphanumeric characters. This property 
   is informational; you cannot change the identifier for existing jobs.
+  
+`model_memory_limit`::
+  (long or string) The approximate maximum amount of memory resources that are 
+  required for analytical processing. Once this limit is approached, data 
```

Comment:
I think the behaviour here differs compared to anomaly detection. If I am not mistaken once the job starts, we calculate how much memory we need and if the limit is below that we will error. Could you confirm this is how memory limits work @tveasey ?

Reply:
Yes this is different for the new analytics. For the first version, this is the approximate maximum amount of RAM that the analytics task will consume. It calculates how much the analysis will need up front and if this quantity is greater than the limit it will fail and report how much memory it needs.